rust/src/client.rs

```diff
@@ -235,6 +236,36 @@ pub(crate) fn client_render_download_progress(
     }
 }
 
+pub(crate) fn microdnf_install(args: Vec<String>) -> Result<()> {
```

Comment:
```suggestion
/// Install packages live into the running root filesystem.
/// This is only intended for use as part of a container builds.
/// For now, we fork/exec microdnf because this is exactly what its
/// main use case is, but in the future we may change the rpm-ostree code
/// to handle this more directly.
pub(crate) fn microdnf_install(args: Vec<String>) -> Result<()> {
```
or so?